Documentation: improvements for the DPF Server getting started section (#675)
pyansys/pydpf-core: docs/source/user_guide/troubleshooting.rst

However, the preceding code disables the reading and generation of the available results for the model.
Any static results that are available for the model are used instead.

Plotting issues
---------------

> Trying to plot a result with DPF plotting capabilities, the following error can be raised:

Reviewer: ```suggestion
When trying to plot a result with DPF, the following error might be raised:
```

.. code-block:: default

    ModuleNotFoundError: No module named 'pyvista'

> Then, to use DPF plotting capabilities, it is needed to install PyVista with:

Reviewer: ```suggestion
In that case, simply install PyVista with:
```
